.github/workflows/npm-package.yml

```diff
@@ -0,0 +1,100 @@
+name: NPM package
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
```

Comment:
instead of branch I would have conan id: `curaengine/5.9.0@_/_` or `curaengine/latest@testing/main`